cfgov/v1/tests/test_auth_forms.py

```diff
@@ -34,3 +34,79 @@ def test_save_without_commit_doesnt_send_email(self, send_email):
         self.assertTrue(form.is_valid())
         form.save(commit=False)
         send_email.assert_not_called()
+
+    def test_duplicate_email_fails_validation(self, send_email):
+        User.objects.create(username='foo', email=self.email)
+        form = UserCreationForm(self.userdata)
+        self.assertFalse(form.is_valid())
+        self.assertTrue(form.errors['email'])
+
+
+class UserEditFormTestCase(TestCase):
```

Comment:
Just a stylistic question: would it not be better to factor out the common stuff like the `data` dict into setUp?